main.py

```diff
@@ -142,6 +143,7 @@ def _load_link(self, link_att):
 
         if link_att['enabled']:
             link.enable()
+            self.link_up.add(link.id)
```

Comment:
For handling link up, we can use execute to periodically poll for when to end the cool down caused by the link up timer. We can then generate interruptions relating to this cool down; entering links into cool down when interrupted, starting the cool down timer when no other interruptions are applied, and ending the cool down with the execute method. 

Additionally, for all methods that affect status within topology we should be having those generate interruptions.

Reply:
@Ktmi, if you could capture another issue for link up timer to also support status_reason similar to https://github.com/kytos-ng/topology/issues/131, and then we can continue the discussion there. Yes, we'll need this cool down following a similar pattern aligned with the timer. Notice though that currently, the cooldown for link up is only for link enabling and when handling link activations the cooldown doesn't restart for other interruptions type like liveness or mw once they go up. This is a relatively old requirement, it might evolve, but currently that's how we have it. It was introduced only to amortize excess of link-flap up notifications.

Reply:
Getting back to the point that I raised though. `self.link_up.add(link.id)` doesn't look coherent here, could you remove it? Initially, all `GenericEntity` will be `active: False` when loaded. 

Other than that I'm also waiting for your feedback on how you locally explored it, and if e2e tests are passing, let me know. After that I can potentially explore it too.